ouroboros-consensus/ouroboros-consensus-mock/test/Test/ThreadNet/TxGen/Mock.hs

```diff
@@ -40,8 +42,12 @@ genSimpleTx addrs u = do
         outs         = if amount == fortune
             then [outRecipient]
             else [outRecipient, (sender, fortune - amount)]
-    return $ Tx ins outs
+    expiry <- (mkExpiry . SlotNo . fromIntegral) <$> generateBetween 2 10
```

Comment:
This expiry doesn't seem to affect anything else, so 2-10 seems reasonable.

Should we also include no expiry here? If I recall correctly, I only introduced the `DoNotExpire` ctor because I wasn't sure how to update the mempool tests, so I side-stepped it that way: expiry in `ThreadNet`, no expiry in those other tests.

@mrBliss Please advise.

Reply:
I don't think it matters that much. I think I would include `DoNotExpire` here too, just to cover everything.

Should the expiry range be dependent on `k`? Although they are unrelated, it might make a difference when we're testing with network partitions and such? I suppose we can keep it [2, 10] now and update it when/if needed.

Reply:
Done, thanks.